files/brews/mongodb.rb

```diff
@@ -6,6 +6,22 @@ class Mongodb < Formula
   stable do
     url "https://fastdl.mongodb.org/src/mongodb-src-r2.6.8.tar.gz"
     sha1 "d67254ef3ba5ba81e21c56be9b919c3a10e01b32"
+
+    # Review this patch with the next stable release.
+    # Note it is a different patch to the one applied to all builds further below.
+    # This is already fixed in the devel & HEAD builds.
+    if MacOS.version == :yosemite
+      patch do
+        url "https://github.com/mongodb/mongo/commit/759b6e8.diff"
```

Comment:
The module installs [MongoDB r2.6.8](https://github.com/intercom/puppet-mongodb/blob/537a85bfb3ecbe2a63b662c96a4b0aa4006472b4/files/brews/mongodb.rb#L7), which already contains this commit (as https://github.com/mongodb/mongo/commit/8ffd49eda26de47acb766b4094f7aba4ddb7a25d).

MongoDB currently compiles successfully through Boxen on Yosemite, I don't think we should mess with it here.